src/content/doc-surrealdb/cli/index.mdx

```diff
@@ -29,7 +29,7 @@ Unless you specify otherwise, the CLI will start a database in memory that serve
 In another window, you can then open up an interactive shell to make queries using the  `surreal sql` command.
 
 ```bash
-surreal sql --endpoint http://localhost:8000 --namespace ns --database db --pretty
+surreal sql --endpoint http://localhost:8000 --namespace ns --database db --auth-level root --username root --password root --pretty
```

Comment:
Thanks for the improvement! One small change: for simplicity the `--auth-level root` part can be taken out because the default level for --username and --password is root. (You'll see --auth-level mostly when logging in as a namespace or database user)

Reply:
@Dhghomon Thanks for the feedback!
Updated the PR.